composer.json

```
@@ -229,7 +229,7 @@
         "symfony/browser-kit": "^5.4",
         "symfony/debug-bundle": "^5.4",
         "symfony/dotenv": "^5.4",
-        "symfony/flex": "^1.7",
+        "symfony/flex": "^1.17",
```

```suggestion
        "symfony/flex": "^1.17.1",
```
According to the blog post.

---

Just to clarify :) https://symfony.com/blog/upgrade-flex-on-your-symfony-projects

---

Yes, exactly this part:
> But if the versions entry is showing a version that **is less than v1.17.1**, then, please keep reading, this blog post is for you.